app/views/kanjis/show.html.erb

```diff
@@ -1,20 +1,52 @@
 <div class="container">
-<%= image_tag "kanji_image.png", alt: "kanji-img", class: "mx-auto d-block w-30 rounded "%>
-<div class="default-card d-flex justify-content-center gap-3 mt-3 col-sm-12">
-<div>
-  <h1 class="display-1"><%= @kanji.character %></h1>
-  </div>
-  <div>
-   <div><span>JLPT Level: </span><h3>N<%= @kanji.jlpt %><% if @kanji.jlpt == 1 || @kanji.jlpt == 2 %><%=  "👑" %><% end %></h3></div>
-    <div><span>Grade: </span><h3><%= @kanji.grade %></h3></div>
+  <%= image_tag "kanji_image.png", alt: "kanji-img", class: "mx-auto d-block rounded w-80"%>
+  <div class="default-card d-flex justify-content-center gap-3 mt-3 col-sm-12">
+    <div><span>JLPT: </span>
+      <h3>N<%= @kanji.jlpt %>
+        <% if @kanji.jlpt == 1 %><%=  " 👹" %>
+        <% elsif @kanji.jlpt == 2 %><%=  " 🕺" %>
+        <% elsif @kanji.jlpt == 3 %><%=  " 😎" %>
+        <% elsif @kanji.jlpt == 4 %><%=  " 😺" %>
+        <% elsif @kanji.jlpt == 5 %><%=  " 🐣" %>
+        <% end %></h3>
```

Comment:
The emoji are fun 👍 
For this kind of pattern, I recommend putting the options in an array and choosing one by the index.

For example:
```
<% rank_emoji = ["", "👹", "🕺", "😎", "😺", "🐣" ] %> #Index 0 is blank, but I think we will label non-jlpt kanji as 0
<%=  " #{rank_emoij[@kanji.jlpt]}" %> #Also included a space so you don't have to type one for each emoji.

```

Reply:
Applied the change! It's much cleaner. Thanks for the suggestion.